app/models/assignment.js

```diff
@@ -55,7 +55,7 @@ export default class AssignmentModel extends Model {
   @computed('tab', 'dcpLupteammemberrole', 'project.{milestones,dcpPublicstatusSimp}')
```

Comment:
@hannahkates this needs to be ` @computed('tab', 'dcpLupteammemberrole', 'project.{milestones,dcpPublicstatus}')`